com.unity.render-pipelines.high-definition/Runtime/Unity.RenderPipelines.HighDefinition.Runtime.asmdef

```diff
@@ -38,6 +40,16 @@
             "name": "com.unity.modules.nvidia",
             "expression": "1.0.0",
             "define": "ENABLE_NVIDIA_MODULE"
+        },
+        {
+            "name" : "com.unity.mathematics",
+            "expression" : "1.2.1",
+            "define" : "ENABLE_MATHEMATICS_1_2_1"
```

Comment:
Here you've added a define but are not using it in the code. Does the define need to exist in the first place? Maybe this is just fine, I'm just wondering what the logic in using these is.

Reply:
I had to add this so the package could be referenced. Not being used in the code.  You know another way to have mathematics referenced without adding this?

Reply:
I'm not sure tbh. If other defines are used in the same way, it's probably fine.

Reply:
should be fine like this